### PR TITLE
feat(nms): Equipmentdashboard - Federation Gateways Cluster Status

### LINKED ISSUE #2842: Add Federation Equipment dashboard and Gateway overview page

### PR DESCRIPTION
**Pull request by ato-dani**
## Summary
This is a draft PR because it is a stacked pr(only the last commit holds the changes relevant to this PR) upon an existing open PR #7804 . Once, that PR is approved, this PR would be changed from draft to a normal PR. A cluster status was added which show the health of both the primary and secondary gateways. It also displays the last fallover time, alongside with the name of the active/primary gateway just like in the design mockups. 

## Test Plan
A test was added under FEGEquipmentGatewayTest.js file which tests the new feature. All previous tests run without any error.

## Additional Information

- [ ] This change is backwards-breaking
Screen Shot 2021-07-01 at 10 03 56 AM

### CODE REVIEW

File: nms/app/packages/magmalte/app/components/context/RefreshContext.js
Comment (andreilee) [inline nms/app/packages/magmalte/app/components/context/RefreshContext.js]: If this doesn't exist under MagmaAPIBindings.js, find a better place to put this constant

**Review by themarwhal — APPROVED** [review]
Approving to unblock, deferring to @andreilee for review. I'll look into why GitHub thinks this PR needs a review from approver-feg today

**Review by andreilee — APPROVED** [review]

**Comment by andreilee** [comment]
Please rebase this

**Pull request merged**
